src/app/lib/analyticsUtils/index.js

```diff
@@ -149,3 +149,45 @@ export const getPublishedDatetime = (attribute, data) => {
     ? getISODate(publishedDatetime)
     : null;
 };
+
+export const getAtiUrl = data => {
+  const cleanedValues = data.filter(({ value }) => value);
+
+  const parsedAtiValues = cleanedValues.map(({ key, value, wrap }) =>
+    wrap ? `${key}=[${value}]` : `${key}=${value}`,
+  );
+
+  return parsedAtiValues.join('&');
+};
+
+export const getPublishingInfo = (service, eventInfo, eventType) => {
```

Comment:
This function doesnt scale very much, is it possible to pass this info down from somewhere, splitting the logic for each type of event tracking